### PR TITLE
58743_user_home

### PR DESCRIPTION
**Pull request by ahammad-reyad**
User home and Profile

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 class StaticPagesController < ApplicationController
   def home
+    if logged_in?
+      @activities = current_user.following_activities.paginate(page: params[:page])
```
Comment (MuhammadTamzid) [inline app/controllers/static_pages_controller.rb]: remove()

File: app/models/user.rb [hunk 2]
```diff
@@ -55,4 +55,7 @@ def following? other_user
     following.include? other_user
   end
 
+  def following_activities
+    Activity.where "user_id in (?)", self.following.pluck(:id)
```
Comment (MuhammadTamzid) [inline app/models/user.rb]: ``self.following.pluck(:id)``
is this work like this

Comment (ahammad-reyad) [inline app/models/user.rb]: NO

Comment (MuhammadTamzid) [inline app/models/user.rb]: OK

**Pull request merged**